InvenTree/part/templates/part/import_wizard/match_references.html

```diff
@@ -0,0 +1,91 @@
+{% extends "part/import_wizard/part_upload.html" %}
+{% load inventree_extras %}
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block form_alert %}
+{% if form.errors %}
+{% endif %}
+{% if form_errors %}
+<div class='alert alert-danger alert-block' role='alert'>
+    {% trans "Errors exist in the submitted data" %}
+</div>
+{% endif %}
+{% endblock form_alert %}
+
+{% block form_buttons_top %}
+    {% if wizard.steps.prev %}
+    <button name="wizard_goto_step" type="submit" value="{{ wizard.steps.prev }}" class="save btn btn-default">{% trans "Previous Step" %}</button>
+    {% endif %}
+    <button type="submit" class="save btn btn-default">{% trans "Submit Selections" %}</button>
+{% endblock form_buttons_top %}
+
+{% block form_content %}
+        <thead>
+            <tr>
+                <th></th>
+                <th>{% trans "Row" %}</th>
+                {% for col in columns %}
+
+                <th>
+                    <input type='hidden' name='col_name_{{ forloop.counter0 }}' value='{{ col.name }}'/>
+                    <input type='hidden' name='col_guess_{{ forloop.counter0 }}' value='{{ col.guess }}'/>
+                    {% if col.guess %}
+                    {{ col.guess }}
+                    {% else %}
+                    {{ col.name }}
+                    {% endif %}
+                </th>
+                {% endfor %}
+            </tr>
+        </thead>
+        <tbody>
+             <tr></tr> {% comment %} Dummy row for javascript del_row method {% endcomment %}
+            {% for row in rows %}
+            <tr {% if row.errors %} style='background: #ffeaea;'{% endif %} part-select='#select_part_{{ row.index }}'>
+                <td>
+                    <button class='btn btn-default btn-remove' onClick='removeRowFromBomWizard()' id='del_row_{{ row.index }}' style='display: inline; float: right;' title='{% trans "Remove row" %}'>
+                            <span row_id='{{ row.index }}' class='fas fa-trash-alt icon-red'></span>
+                    </button>
+                </td>
+                <td>
+                {% add row.index 1 %}
+                </td>
+                {% for item in row.data %}
+                <td>
+                    {% if item.column.guess %}
+                        {% with row_name=item.column.guess|lower %}
+                        {% for field in form.visible_fields %}
+                            {% if field.name == row|keyvalue:row_name %}
```

Comment:
Nice :+1:

Reply:
I think it will be easier to keep the templates like they are , will not be touching existing stuff.